samples/go/demo-server/web_server.go

```diff
@@ -171,6 +171,7 @@ func corsHandle(f httprouter.Handle) httprouter.Handle {
 		// Can't use * when clients are using cookies.
 		w.Header().Add("Access-Control-Allow-Origin", r.Header.Get("Origin"))
 		w.Header().Add("Access-Control-Allow-Methods", "GET, POST")
+		w.Header().Add("Access-Control-Allow-Headers", "Authorization")
```

Comment:
I think that the allowed headers are normally listed rather than put into separate header lines: https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Access-Control-Allow-Headers. I'm not sure if it makes a difference.